src/apps/AppCenter/InstalledApps/AppCardContent.js

```diff
@@ -43,17 +43,10 @@ const AppCardContent = ({ repo, onOpen }) => {
             css={`
               display: flex;
               justify-content: center;
-              margin-bottom: 12px;
+              margin-bottom: 8px;
```

Comment:
```suggestion
              margin-bottom: ${1 * GU}px;
```

Reply:
Haha yes, I wanted to avoid doing too much since we'll be doing this work once we start working on the App Center anyway.